vcpkg.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "csound",
+    "version-string": "6.16.2",
+    "dependencies": [
+        "eigen3",
```

Comment:
Are these libaries being installed as static libraries for linking?

Reply:
Whether they're static or dynamic is based on the triplet. We have a custom triplet file "x64-windows-csound" that installs some as static and some as dynamic. We can/must have a custom triplet for each platform.